rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,712 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere">
```

Comment:
I would recommend moving these rules to a JMS specific file. Something like "websphere-jms.windup.xml" would be an improvement.